### ISSUE TITLE
unresolved dependency: com.github.tototoshi#play-joda-routes-binder_2.11;0.1.0: not found

### ISSUE THREAD

**Issue by NoriakiHoriuchi** [tototoshi/play-joda-routes-binder#4]
Hi, I tried to use this fantastic library in Play 2.4.2 / Scala 2.11.6 / sbt 0.13.8, but got an error below.  What should I do?

```
Error:Error while importing SBT project:
...
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1310)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:235)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[error] sbt.ResolveException: unresolved dependency: com.github.tototoshi#play-joda-routes-binder_2.11;0.1.0: not found
```

**Comment by tototoshi** [comment]
0.1.0 has not released for 2.11.x. 
I will release newer version.

**Comment by tototoshi** [comment]
Published 1.0.0-SNAPSHOT to sonatype. Please try it.

https://oss.sonatype.org/content/repositories/snapshots/com/github/tototoshi/play-joda-routes-binder_2.11/1.0.0-SNAPSHOT/

**Comment by NoriakiHoriuchi** [comment]
Thank you for your 光の速さ commit.
I still fail to resolve dependency (sbt says "not found"), so later will try again.

**Comment by tototoshi** [comment]
Did you add the sonatype snapshot repository to resolvers?

```scala
resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
```

**Comment by NoriakiHoriuchi** [comment]
Oops. I did it!

**Comment by tototoshi** [comment]
Released 1.0.0

**Comment by NoriakiHoriuchi** [comment]
Thanks a lot!
